website/src/app/documentation/demos/page-layouts/page-layouts.demo.html

```diff
@@ -76,6 +85,12 @@ <h4>Demo</h4>
           <span class="nav-text">Command bar</span>
         </a>
       </div>
+      <div>
+        <cds-icon shape="display"></cds-icon>&nbsp;
+        <a class="nav-link" routerLink="/full-page-layouts/action-panel" routerLinkActive="active">
+          <span class="nav-text">Action panel</span>
```

Comment:
pls add some documentation here on how to use the action panel, as currently there is no copy&paste section for it